enterprise/web.jsf20/nbproject/project.properties

```diff
@@ -17,6 +17,6 @@
 
 javac.compilerargs=-Xlint -Xlint:-serial
 javac.source=1.7
-release.external/javax.faces-2.2.7.jar=modules/ext/jsf-2_2/javax.faces.jar
-release.external/javax.faces-2.2.7-license.txt=modules/ext/jsf-2_2/license.txt
-spec.version.base=1.39.0
+release.external/javax.faces-2.3.9.jar=modules/ext/jsf-2_2/javax.faces.jar
+release.external/javax.faces-2.3.9-license.txt=modules/ext/jsf-2_2/license.txt
+spec.version.base=1.38.0
```

Comment:
Spec version needs to be incremented, not decremented (updates are determined by this)

Reply:
Thanks and I apologize for the late response and oversight...I've increased the spec version.